### PR TITLE
Fix sigalarm for windows

### PR DESCRIPTION
**Pull request by irux**

### CODE REVIEW

File: kpops/component_handlers/kafka_connect/timeout.py
Comment (raminqaf) [inline kpops/component_handlers/kafka_connect/timeout.py:27]: Remove old commented code please

File: kpops/component_handlers/kafka_connect/timeout.py
Comment (raminqaf) [inline kpops/component_handlers/kafka_connect/timeout.py:19]: Can you give this variable a more meaningful name please?

**Review by raminqaf — CHANGES_REQUESTED** [review]

**Review by raminqaf — CHANGES_REQUESTED** [review]

**Comment by irux** [comment]
Yes! for example now all the connector test that depends on the timeout function works https://github.com/bakdata/kpops/blob/fix/windows/tests/component_handlers/kafka_connect/test_connect_wrapper.py#L122

**Review by sujuka99 — APPROVED** [review]
LGTM!

Please wait for another approval, too.

**Review by philipp94831 — APPROVED** [review]

**Review by disrupted — CHANGES_REQUESTED** [review]

File: kpops/component_handlers/kafka_connect/timeout.py
Comment (irux) [reply kpops/component_handlers/kafka_connect/timeout.py]: @disrupted done!

**Pull request merged**
